postgres-appliance/scripts/configure_spilo.py

```diff
@@ -612,6 +623,15 @@ def get_placeholders(provider):
 
     placeholders['BGMON_LISTEN_IP'] = get_listen_ip()
 
+    if 'SSL_CA' in placeholders and placeholders['SSL_CA_FILE'] == '':
+        placeholders['SSL_CA_FILE'] = os.path.join(placeholders['RW_DIR'], 'certs', 'ca.crt')
+    if not 'SSL_CA' in placeholders and placeholders['SSL_CA_FILE'] != '':
+        placeholders['SSL_CA_FILE'] = ''
```

Comment:
This one should be removed. It is quite common that people just mount certificates and provide their location in the `*_FILE` env var.

Reply:
ok as you prefer, it was just to solve a possible inconsistency in input parameters

Reply:
fix requested done